test/functional/apps/visualize/_tile_map.js

```diff
@@ -45,6 +45,30 @@ export default function ({ getService, getPageObjects }) {
       });
     });
 
+
+    /**
+     * manually compare data due to possible small difference in numbers. This is browser dependent.
+     */
+    function compareTableData(expected, actual) {
+
+      expect(actual.length).to.eql(expected.length);
+
+      function tokenize(row) {
+        const tokens = row.split(' ');
+        return {
+          geohash: tokens[0],
+          count: tokens[1],
+          lat: Math.floor(parseFloat(tokens[4])),
+          lon: Math.floor(parseFloat(tokens[6]))
+        };
+      }
+
+      const act = JSON.stringify(actual.map(tokenize));
```

Comment:
Can we use [.eql](https://github.com/Automattic/expect.js#api) here instead of JSON.stringify?
```
expect({ a: 'b' }).to.eql({ a: 'b' });
```